src/components/main/Main.tsx

```diff
@@ -93,12 +84,8 @@ const Main = ({ children }: Props): JSX.Element => {
           id={APP_NAVIGATION_PLATFORM_SWITCH_ID}
           selected={Platform.Builder}
           platformsProps={platformProps}
-          color={
-            isMobile ? theme.palette.primary.main : theme.palette.secondary.main
```

Comment:
Why are we removing the isMobile here?

Reply:
Because the background is always white and the secondary color is white so it wouldn't be readable on mobile.